nomad/node_endpoint.go

```diff
@@ -52,6 +52,23 @@ type Node struct {
 	updatesLock sync.Mutex
 }
 
+func (n *Node) EmitEvent(args *structs.EmitNodeEventRequest, reply *structs.EmitNodeEventResponse) error {
```

Comment:
Should probably be EmitEvents

Reply:
Same for EmitNodeEvent**s**Request and EmitNodeEvent**s**Response too?